isso/tests/test_vote.py

```diff
@@ -1,29 +1,27 @@
-
 from __future__ import unicode_literals
 
-import os
 import json
 import tempfile
 import pkg_resources
 import unittest
 
 from werkzeug.wrappers import Response
 
-from isso import Isso, core, config, dist
+from isso import Isso, core, config
 from isso.utils import http
 
 from fixtures import curl, loads, FakeIP, JSONClient
+
 http.curl = curl
 
 
 class TestVote(unittest.TestCase):
-
     def setUp(self):
         self.path = tempfile.NamedTemporaryFile().name
 
     def makeClient(self, ip):
 
-        conf = config.load(pkg_resources.resource_filename('isso', 'defaults.ini'))
```

Comment:
Please don't include these unrelated formatting changes - they make the PR harder to review.